### PR TITLE
Add schedule bodkia

### PR DESCRIPTION
**Pull request by bodkia22**

### CODE REVIEW

File: src/app/components/add-class-form/add-class-form.component.ts [hunk 1]
```diff
@@ -47,7 +49,7 @@ export class AddClassFormComponent implements OnInit {
   toggleForm = () => this.formIsOpen = !this.formIsOpen;
 
   getTeachersToSelectData = () => {
-    this.teacherService.getTeachersToSelect().subscribe(data => this.teachers = data)
+    this.teacherService.getClassTeachersToSelect().subscribe(data => this.teachers = data)
   }
```
Comment (DmMov) [inline src/app/components/add-class-form/add-class-form.component.ts]: Why class teachers?

File: src/app/services/teachers/teacherinfo.service.ts [hunk 2]
```diff
@@ -23,8 +23,11 @@ export class TeacherinfoService {
...
-  getTeachersToSelect = ():Observable<ITeacherToSelect[]>=>{
+  getClassTeachersToSelect = (): Observable<ITeacherToSelect[]>=>{
     this.httpOptions.loadHeaders();
-    return this.http.get<ITeacherToSelect[]>(this.url + "/GetTeacherBySchoolShort", this.httpOptions.options);
+    return this.http.get<ITeacherToSelect[]>(this.url + "/GetTeachersBySchoolShort", this.httpOptions.options);
   }
+
+  
+  
 }
```
Comment (DmMov) [inline src/app/services/teachers/teacherinfo.service.ts]: You know what to do

File: src/app/components/add-class-form/add-class-form.component.ts (hunk 1 above)
Comment (Hrabarchuk) [reply src/app/components/add-class-form/add-class-form.component.ts]: > Why class teachers?

Maybe  , when you create class , you must select a Class Teacher?
But you get all teachers but not a class teachers)))))

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import { HeadAssistantScheduleComponent } from './pages/head-assistant-schedule/head-assistant-schedule.component';
```
Comment (Redingston) [inline src/app/app-routing.module.ts]: it's should be in app-routing module

File: src/app/app-routing.module.ts [hunk 4]
```diff
@@ -165,7 +167,5 @@ export const RoutingComponents = [
   PupilProfileComponent,
   HeadAssistantProfileComponent,
   HeadAssistantCabinetComponent,
-  ScheduleComponent,
-  LessonsComponent,
-  JournalColumnComponent
```
Comment (Redingston) [inline src/app/app-routing.module.ts]: Why you deleted this export?

File: src/app/components/add-schedule-form/add-schedule-form.component.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  </mat-form-field>
+
+  <yps-form-field 
+  *ngFor="let field of fields" 
+  [field]="field" 
+  [formControlName]="field.name"
+  
+  >
+  </yps-form-field>
+
```
Comment (DmMov) [inline src/app/components/add-schedule-form/add-schedule-form.component.html]: Fix this.

File: src/app/pages/head-assistant-schedule/head-assistant-schedule.component.html [hunk 6]
```diff
@@ -0,0 +1,2 @@
+<div style="background-color: azure;">Schedule info</div>
```
Comment (DmMov) [inline src/app/pages/head-assistant-schedule/head-assistant-schedule.component.html]: What is this style for?

**Review by Redingston — DISMISSED** [review]

**Review by Redingston — APPROVED** [review]

**Review by DmMov — APPROVED** [review]

**Review by vunt1k — APPROVED** [review]

**Pull request merged**